modules/plugin/coverage-multidim/netcdf/src/test/resources/org/geotools/coverage/io/netcdf/test-data/netcdf.projections.properties

```diff
@@ -2,3 +2,5 @@
 971802=PROJCS["lambert_conformal_conic_2SP", GEOGCS["unknown", DATUM["unknown", SPHEROID["unknown", 6377397.0, 299.15550239234693]], PRIMEM["Greenwich", 0.0], UNIT["degree", 0.017453292519943295], AXIS["Geodetic longitude", EAST], AXIS["Geodetic latitude", NORTH]], PROJECTION["Lambert_Conformal_Conic_2SP"], PARAMETER["central_meridian", 13.333333015441895], PARAMETER["latitude_of_origin", 46.0], PARAMETER["standard_parallel_1", 46.0], PARAMETER["standard_parallel_2", 49], PARAMETER["false_easting", 0.0], PARAMETER["false_northing", 0.0], UNIT["m", 1.0], AXIS["Easting", EAST], AXIS["Northing", NORTH], AUTHORITY["EPSG","971802"]]
 971835=PROJCS["albers_conical_equal_area", GEOGCS["unknown", DATUM["unknown", SPHEROID["unknown", 6378137.0, 298.2572221010042]], PRIMEM["Greenwich", 0.0], UNIT["degree", 0.017453292519943295], AXIS["Geodetic longitude", EAST], AXIS["Geodetic latitude", NORTH]], PROJECTION["Albers_Conic_Equal_Area"], PARAMETER["central_meridian", -126.0], PARAMETER["latitude_of_origin", 45.0], PARAMETER["standard_parallel_1", 50.0], PARAMETER["false_easting", 1000000.0], PARAMETER["false_northing", 0.0], PARAMETER["standard_parallel_2", 58.5], UNIT["m", 1.0], AXIS["Easting", EAST], AXIS["Northing", NORTH], AUTHORITY["EPSG","971835"]]
 971807=PROJCS["Lambert_Conformal_Conic", GEOGCS["GCS_unnamed ellipse", DATUM["D_unknown", SPHEROID["Unknown", 6370000.0, 0.0]], PRIMEM["Greenwich", 0.0], UNIT["degree", 0.017453292519943295]], PROJECTION["Lambert_Conformal_Conic"], PARAMETER["central_meridian", 12.0], PARAMETER["latitude_of_origin", 50.0], PARAMETER["standard_parallel_1", 30], PARAMETER["false_easting", 0.0], PARAMETER["false_northing", 0.0], PARAMETER["scale_factor", 1.0], PARAMETER["standard_parallel_2", 60], UNIT["m", 1.0], AXIS["Easting", EAST], AXIS["Northing", NORTH], AUTHORITY["EPSG","971807"]]
+970916=PROJCS["polar_stereographic",GEOGCS["unknown",DATUM["unknown",SPHEROID["unknown", 6371229.0, 0.0]],PRIMEM["Greenwich", 0.0],UNIT["degree", 0.017453292519943295],AXIS["Geodetic longitude", EAST],AXIS["Geodetic latitude", NORTH]],PROJECTION["Polar_Stereographic"],PARAMETER["central_meridian", -135.0],PARAMETER["latitude_of_origin", 90.0],PARAMETER["scale_factor", 0.9330127239227295],PARAMETER["false_easting", 0.0],PARAMETER["false_northing", 0.0],UNIT["m", 1.0],AXIS["Easting", EAST],AXIS["Northing", NORTH],AUTHORITY["EPSG","970916"]]
+970917=PROJCS["polar_stereographic",GEOGCS["unknown",DATUM["unknown",SPHEROID["unknown", 6371229.0, 0.0]],PRIMEM["Greenwich", 0.0],UNIT["degree", 0.017453292519943295],AXIS["Geodetic longitude", EAST],AXIS["Geodetic latitude", NORTH]],PROJECTION["Polar_Stereographic"],PARAMETER["central_meridian", -135.0],PARAMETER["latitude_of_origin", 90.0],PARAMETER["scale_factor", 0.9330127239227295],PARAMETER["false_easting", 0.0],PARAMETER["false_northing", 0.0],UNIT["km", 1000.0],AXIS["Easting", EAST],AXIS["Northing", NORTH],AUTHORITY["EPSG","970917"]]
```

Comment:
Could be interesting to have a case in feet to test the generality of the approach.

Reply:
I don't have any dataset at hand with a reference with feet or a definition of that.
I might have to build a fake one.